tests/system/python/api/test_audit.py

```diff
@@ -79,20 +79,34 @@ def test_default_get_audit(self, fledge_url, wait_time, request_params, total_co
         assert 200 == r.status
         r = r.read().decode()
         jdoc = json.loads(r)
+
+        elems = jdoc['audit']
+
+        if storage_plugin == 'postgres':
+            # sqlite has an additional entry: CONAD, INFORMATION, '{{"name":"sqlite", ...}'
```

Comment:
Why do the different plugins have different audit entries?

Reply:
@MarkRiddoch Storage category when with SQLite plugin, creates a child category `sqlite'; hence an additional CONAD entry. I'm not sure if when storage plugin is postgres there should be a child category for the same?